test/system/inspect.spec.ts

```diff
@@ -924,21 +937,6 @@ describe('inspect', () => {
           },
         ],
       },
-      {
```

Comment:
This needs to be removed as Jinja2 always gets installed with the latest version for this testcase. It's not sustainable to update the testcase with the latest version always.
Can be moved to another PR.